app/views/agents/_form.html.haml

```diff
@@ -9,21 +9,19 @@
   = hidden_field_tag :deletable, deletable if deletable
   = hidden_field_tag agent_field_name(:id, name_prefix), agent.id if agent.id
 
-  - if affiliation?(agent) &&  params[:parent_id]
```

Comment:
why removing this 2 lines?, it used to save the type of agent when the created agent is an affiliation (an agent of an agent)